.gitmodules

```diff
@@ -5,3 +5,7 @@
 [submodule "test/tests"]
 	path = test/jsontests
 	url = https://github.com/ethereum/tests.git
+[submodule "hera"]
+	path = hera
+	url = https://github.com/ewasm/hera
```

Comment:
CI cannot find the commit 60963e52d20247376b95e444982453f95a5b300a.
Also, don't use `branch`. It is just a hint for users, and will probably change to master soon.

Reply:
Will check out the correct hash once we merge cmake into hera.

Reply:
Oh, it works now :)